frontend/src/sagas/event.js

```diff
@@ -102,14 +102,9 @@ function connectTo(url) {
     return new Promise(resolver.bind(socket))
 }
 
-function* events() {
-    const protocol = location.protocol === "https:" ? "wss:" : "ws:";
-    const url = `${protocol}//${location.host}/ws`;
-
+function* events(socket) {
     while (true) {
         try {
-            console.log(`Connecting to backend: ${url}`)
-            const socket = yield call(connectTo, url)
```

Comment:
I understand why you moved the code but did you consider that it will not retry to connect to the backend on startup while it did that before?